content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md

```diff
@@ -11,10 +11,19 @@ registration_url: https://www.eventbrite.com/e/93157648033
 captions: https://www.captionedtext.com/client/event.aspx?EventID=4341007&CustomerID=321
 
 # start date
-date: 2020-03-25 14:00:00 -0500
+date: 2020-03-25 15:00:00 -0500
```

Comment:
Was this time also changed intentionally? Curious to know if the workflow tool accidentally changed it on you.